starters/next-drupal-starter/pages/[...alias].jsx

```diff
@@ -182,8 +182,6 @@ export async function getStaticProps(context) {
       objectName: resourceName,
       id: uuid,
       params: context.preview ? previewParams : params,
-      // if previewing a revision, force a fetch to Drupal
```

Comment:
I made a similar change in my SSR work, so this will likely conflict. Not a big deal though - should be easy to resolve.